docs/swagger.yaml

```diff
@@ -105,88 +112,128 @@ paths:
                       }
                     ]
                   }
+        default:
+          description: Default problem response
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/Problem"
 
-  "/report/v1/ledgertransactions":
+  "/report/v2/ledgers/{LedgerId}/{Account}/dates/{LedgerDate}":
```

Comment:
The {Account} part of this url does not look right.

Reply:
We should use camelCase for path parameters, as specified in the RFC: https://github.com/vippsas/rfc/blob/master/docs/0001-api-guidelines.md#use-camelcase